server/protocols/jmap/src/main/java/org/apache/james/jmap/model/SetError.java

```diff
@@ -57,6 +57,31 @@ public Builder type(String type) {
             return this;
         }
 
+        public Builder maxQuotaReached() {
+            this.type = "maxQuotaReached";
```

Comment:
I don't see that as a good solution : you don't prevent type to be defined twice and you don't make it clear maxQuotaReached is setting a type the same way invalidArguments do (for example).
Either you change the returned type to BuilderWithType to be sure that we can't set a type twice or you take an enum value in `type()` method